package cn.exception;

public class ExceptionTest {


    public static void main(String[] args) {


        System.out.println(MyMath.div(100, 0));

    }
}
